prime-router/src/main/kotlin/cli/tests/TestReportStream.kt

```diff
@@ -477,7 +477,8 @@ abstract class CoolTest {
                 from covid_result_metadata as cr
                 where cr.report_id = ?"""
             val ret = ctx.fetch(sql, reportId)?.into(Int::class.java)
-            passed = ret?.size > 0
+            val sz = ret?.size
+            passed = sz != null && sz > 0
```

Comment:
Instead of the `sz` intermediate value, suggest one line of `passed = ret != null && ret.size > 0`

Reply:
Updated